### ISSUE TITLE
KyrdaDB API key error

### ISSUE THREAD

**Issue by zim514** [rmrector/script.artwork.beef#41]
With kyradb_userkey set to a correct value for my KyraDB account and debug logs on, Invalid API key errors still get thrown.

Setting apikey.kyradb to the value from my account instead of using the default value in projectkeys.py seems to fix it.

Kyradb seems to require the API and userkey to match. 

I'd just move the API key setting out of the "emergency" section to the "advanced" so that users set it along with the userkey, but I'm not sure if that's how KyraDB is supposed to work. I don't know why they would use two keys in this way so I didn't submit a PR with my idea

**Comment by jangrewe** [comment]
I have a similar problem, and i don't even use KyraDB:
```
WARNING: [script.artwork.beef] Error with provider 'KyraDB.com': KyraDB User key is required for artwork from KyraDB: ('animatedposter', 'animatedkeyart', 'animatedfanart', 'characterart')
```

@rmrector: If an User Key is (now?) required, i'd like to see an option to just disable lookups at KyraDB. :-)

**Comment by rmrector** [comment]
Ya, I just got confirmation that KyraDB _does_ use two keys that way. I'll change that in the next update.

@jangrewe unrelated. Disable animated artwork and characterart for movies.

**Comment by rmrector** [comment]
Fixed in 0.26.0
